### PR TITLE
Renaming HMRC plugin to UK

### PR DESCRIPTION
**Pull request by allenbross-wf**
#### Description of change

Renamed HMRC plugin to more appropriate UK name.
Created alias for HMRC for backward compatibility. 

#### Steps to Test

CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/CntlrCmdLine.py [hunk 1]
```diff
@@ -175,8 +175,8 @@ def parseArgs(args):
                       help=_("Specify a disclosure system name and"
                              " select disclosure system validation.  "
                              "Enter --disclosureSystem=help for list of names or help-verbose for list of names and descriptions. "))
-    parser.add_option("--hmrc", action="store_true", dest="validateHMRC",
-                      help=_("Select U.K. HMRC disclosure system validation."))
```
Comment (derekgengenbacher-wf) [inline arelle/CntlrCmdLine.py:179]: Do we want to get rid of this? It seems like we would want to keep this around a for a bit to allow people to switch over to the new UK option without breaking them?

Comment (aaroncameron-wk) [reply arelle/CntlrCmdLine.py:179]: `--hmrc` is still available, `--uk` is just another option for the same arg.

**Review by aaroncameron-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

File: docs/source/plugins/popular/validation.md [hunk 2]
```diff
@@ -91,12 +91,12 @@ SEC at [StructuredData@sec.gov](mailto:StructuredData@sec.gov).
 :::{autodoc2-docstring} arelle.plugin.validate.FERC
 :::
 
-## Validate HMRC
+## Validate UK
 
 :::{index} Validate HMRC
```
Comment (aaroncameron-wk) [inline docs/source/plugins/popular/validation.md:96]: Sneaky one here

File: pyproject.toml [hunk 3]
```diff
@@ -336,6 +336,7 @@ module = [
     'arelle.plugin.validate.FERC',
     'arelle.plugin.validate.HMRC',
```
Comment (aaroncameron-wk) [inline pyproject.toml:337]: This module doesn't exist anymore, so can be removed

**Review by aaroncameron-wk — APPROVED** [review]

**Review by aaroncameron-wk — APPROVED** [review]

**Pull request merged**